apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -23,6 +23,7 @@ li.pl-order-blocks-code {
   margin-bottom: 0px !important;
 }
 
+
```

Comment:
I would prefer to not add this unneccessary extra white space

Reply:
Oops, I think this got in there by mistake when I was copying and cutting things while changing the CSS.